integration/conftest.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+import fakeredis
+import redis as r
+import pytest
+
+from storey import RedisDriver
+
+REDIS_ENDPOINT = os.environ.get('REDIS_URL')
+
+
+@pytest.fixture()
+def redis():
+    if REDIS_ENDPOINT:
+        yield r.Redis.from_url(REDIS_ENDPOINT)
+    else:
+        yield fakeredis.FakeRedis(decode_responses=True)
+
+
+@pytest.fixture()
+def redis_key_prefix():
+    yield "storey-test:"
```

Comment:
Could we create a database for the tests rather than rely on a prefix?

Reply:
Actually there is no 'database' in redis:
https://stackoverflow.com/questions/35621324/how-to-create-own-database-in-redis#:~:text=You%20don't%20create%20a,between%20the%20databases%2C%20call%20SELECT%20.&text=Show%20activity%20on%20this%20post.,-Use%20select%20%2C%20for

Reply:
We decided to maybe consider this later.